oracle/README.md

```diff
@@ -33,14 +35,28 @@ The Oracle check requires either access to the `cx_Oracle` Python module, or the
    sudo ldconfig
    ```
 
-2. Decompress those libraries in a given directory available to all users on the given machine (i.e. `/opt/oracle`):
+2. Decompress both packages into a single directory that is available to all users on the given machine (for example, `/opt/oracle`):
 
    ```shell
    mkdir -p /opt/oracle/ && cd /opt/oracle/
    unzip /opt/oracle/instantclient-basic-linux.x64-12.1.0.2.0.zip
    unzip /opt/oracle/instantclient-sdk-linux.x64-12.1.0.2.0.zip
    ```
 
+###### Windows
+1. Install the [Microsoft Visual Studio 2017 Redistributable][17] or the appropriate version for the Oracle Instant Client installation.
+
+2. Go to the [download page][4] and install both the *Instant Client Basic* and *SDK* packages.
+
+3. Extract both packages into a single directory that is available to all users on the given machine (for example, `C:\oracle`):
+
+4. Add the System Variable `OCI_LIB64=C:\oracle` where the value is the path to the directory containing the Oracle Instant Client.
+
+5. Modify the `PATH` environment variable to contain the path of the Oracle Instant Client.
```

Comment:
I'm not sure if setting the PATH and the OCI-LIB64 vars is a requirement. Have you checked?

Reply:
I saw that in a few tutorials and in one of the linked support tickets.

Reply:
My point is that I don't think that setting the PATH environment variable on windows has any effect on the agent as it runs under a different user.

Reply:
Ah, I was just including the complete installation steps for Windows Instant client. Perhaps its simpler to just refer to the instructions on Oracle instead of explicitly detailing them here?